sphinx/writers/latex.py

```diff
@@ -601,6 +601,11 @@ def __init__(self, document, builder):
                     (',numfigreset=%s' % self.numfig_secnum_depth)
             else:
                 self.elements['sphinxpkgoptions'] += ',nonumfigreset'
+            try:
+                if builder.config.math_numfig:
+                    self.elements['sphinxpkgoptions'] += ',mathnumfig'
+            except AttributeError:
+                pass
```

Comment:
I have corrected the bare except but I would prefer more direct way to access the mathbase extension ``math_numfig`` config value

Reply:
Ideally, this code should move to the extension. This is sphinx-core.
And we should provide a way to enhance LaTeX output.

But, as I proposed in #4333, math feature should be merged to sphinx-core in future.
So I'm okay this as is.